src/main/java/xyz/nucleoid/plasmid/game/config/GameConfigs.java

```diff
@@ -80,6 +80,11 @@ private static Identifier identifierFromPath(Identifier location) {
         return CONFIGURED_GAMES.get(identifier);
     }
 
+    @Nullable
+    public static Identifier getIdentifier(ConfiguredGame<?> game) {
```

Comment:
This shouldn't be provided in the API, given not all games have identifiers and we are currently trying to move away from using them as much as possible.

Reply:
Oops, this was unintentional.